src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/pact/PactContractTest.kt

```diff
@@ -29,56 +29,47 @@ class PactContractTest {
   @State("Server is healthy")
   fun `ensure server is healthy`() {}
 
-  @State("Participations exist for a person with prison number B1234BB")
-  fun `ensure participations exist for a person with prison number B1234BB`() {}
+  @State("A participation can be created")
+  fun `ensure a participation can be created`() {}
 
-  @State("A person exists with prison number A1234AA")
-  fun `ensure a person exists with prison number A1234AA`() {}
+  @State("Course d3abc217-75ee-46e9-a010-368f30282367 exists")
+  fun `ensure course d3abc217-75ee-46e9-a010-368f30282367 exists`() {}
 
-  @State("Courses exist on the API")
-  fun `ensure courses exist on the API`() {}
+  @State("Course d3abc217-75ee-46e9-a010-368f30282367 has offerings 790a2dfe-7de5-4504-bb9c-83e6e53a6537 and 7fffcc6a-11f8-4713-be35-cf5ff1aee517")
+  fun `ensure course d3abc217-75ee-46e9-a010-368f30282367 has offerings 790a2dfe-7de5-4504-bb9c-83e6e53a6537 and 7fffcc6a-11f8-4713-be35-cf5ff1aee517`() {}
 
-  @State("Course names exist on the API")
-  fun `ensure course names exist on the API`() {}
+  @State("Course participation 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 exists")
+  fun `ensure course participation 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 exists`() {}
 
-  @State("A course exists with ID d3abc217-75ee-46e9-a010-368f30282367")
-  fun `ensure a course exists with ID d3abc217-75ee-46e9-a010-368f30282367`() {}
+  @State("Courses d3abc217-75ee-46e9-a010-368f30282367, 28e47d30-30bf-4dab-a8eb-9fda3f6400e8, and 1811faa6-d568-4fc4-83ce-41118b90242e and no others exist")
+  fun `ensure courses d3abc217-75ee-46e9-a010-368f30282367, 28e47d30-30bf-4dab-a8eb-9fda3f6400e8, and 1811faa6-d568-4fc4-83ce-41118b90242e and no others exist`() {}
```

Comment:
Question: Is this the same as delete all courses?

Reply:
No, we only have the ability to delete individual course participations